content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -281,7 +324,22 @@ and installing the sidecar injector webhook configuration on the remote cluster
     EOF
     {{< /text >}}
 
-    Then, apply the Istio configuration on the external cluster:
+1.  If you are using an IP address for the `EXTERNAL_ISTIOD_ADDR`, instead of a proper DNS hostname,
+    delete the proxy metadata and webhook config environment variables from the configuration:
+
+    {{< warning >}}
+    This is not recommended in a production environment.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ sed  -i'.bk' \
+      -e '/proxyMetadata:/,+2d' \
+      -e '/INJECTION_WEBHOOK_CONFIG_NAME/,+1d' \
+      -e '/VALIDATION_WEBHOOK_CONFIG_NAME/,+1d' \
```

Comment:
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/19339970/180218536-ca4b458a-5405-4b4c-aaff-262fd53d205f.png">  
keep looping of the process of  istiod in istiod between istio-system and  external-istiod. 

BTW,We need to change the  VALIDATION_WEBHOOK_CONFIG_NAME and INJECTION_WEBHOOK_CONFIG_NAME  to  istio-sidecar-injector-${namespace} istio-validator-${namespace}

Reply:
Thanks, good catch. I'll fix this in a followup PR. Interesting that the integration test didn't catch this bug.

Reply:
ok. thanks for you pr.